### DISCUSSION TITLE
search user comments on records.

### DISCUSSION THREAD

**Discussion by thinkbig1979** [teableio/teable#1061] [Q&A]
Is there a way to search/filter on the contents of the comments that can be added to records by users?

**Comment by caoxing9** [comment]
You mean is that search in comment list?

**Comment by thinkbig1979** [comment]
When you open up a record and get the modal form view, there is a conversation section on the right of that form where users can add comments about the record. These comments are currently not searchable, not even from global search, but sometimes there is valuable information in there, so it would be great to incorporate them into the search function somehow.
